<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>971</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgWidget{
background-color:qconicalgradient(cx:1, cy:0.170455, angle:0, stop:0 rgba(237, 212, 0, 255), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Ubuntu&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>AddBooks</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>160</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="book_id_label">
       <property name="text">
        <string>Book ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="book_title_label">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="book_author_label">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="book_genre_label">
       <property name="text">
        <string>genre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="book_price_label">
       <property name="text">
        <string>price</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>268</y>
      <width>261</width>
      <height>175</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="id_edit">
       <property name="styleSheet">
        <string notr="true">background-color:gba(0,0,0,0);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="title_edit">
       <property name="styleSheet">
        <string notr="true">background-color:gba(0,0,0,0);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="author_edit">
       <property name="styleSheet">
        <string notr="true">background-color:gba(0,0,0,0);
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="genre_edit">
       <property name="styleSheet">
        <string notr="true">background-color:gba(0,0,0,0);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="price_edit">
       <property name="styleSheet">
        <string notr="true">background-color:gba(0,0,0,0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>441</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submit_button">
       <property name="styleSheet">
        <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(224, 110, 28, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refresh_button">
       <property name="styleSheet">
        <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(224, 110, 28, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>refresh the App</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 87, 83);
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
